README.md

```diff
@@ -14,8 +17,12 @@ Ipywidgets lets you create interactive widgets in your notebook.
 Installation is fairly straightforward. You install the python package,
 then enable the extension.
 
-## python-markdown
-The `python-markdown` extension is useful for interweaving computational
-cells (e.g., python cells) and markdown cells. As this extension automatically
-runs code in the notebook, you need to be sure to "trust" the notebooks in your
-`postBuild` script (see the script in this repo for example).
+JupyterLab requires installing a jupyter labextension.
+
+## postBuild
```

Comment:
In this section perhaps you could just put something like

> The postBuild file defines commands (one per line) to be run with bash.
> In this case, we use it to install a Jupyter Lab extension
> (by calling `jupyter labextension`) which allows `ipywidgets`
> to be displayed within notebooks.

Reply:
ok.

Reply:
Can you just remove the `## postBuild` section (and its content) totally from this PR? I think it might confuse folks who are just trying to figure out how to install jupyterlab extensions

Reply:
Done.